<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlPropertyGridWidget</class>
 <widget class="QWidget" name="ControlPropertyGridWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupWidgetFrame">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>1</y>
     <width>299</width>
     <height>200</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLineEdit" name="tagLineEdit">
    <property name="geometry">
     <rect>
      <x>52</x>
      <y>90</y>
      <width>241</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>35</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Class:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>65</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Name:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="objectNameLineEdit">
    <property name="geometry">
     <rect>
      <x>52</x>
      <y>60</y>
      <width>241</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="classNameLineEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>52</x>
      <y>30</y>
      <width>241</width>
      <height>22</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>95</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tag:</string>
    </property>
   </widget>
   <widget class="QFrame" name="frameCustomControlData">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>300</width>
      <height>81</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>211</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Custom Control Class Name:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="customControlLineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>281</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnResetMorphToCustomControl">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>45</y>
       <width>180</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Reset Custom Control</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QFrame" name="frameMorphToCustomControl">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>200</y>
     <width>300</width>
     <height>31</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <widget class="QPushButton" name="btnMorphToCustomControl">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>0</y>
      <width>216</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Morph Into Custom Control</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>classNameLineEdit</tabstop>
  <tabstop>objectNameLineEdit</tabstop>
  <tabstop>tagLineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
